Python-Using-Pip.md

```diff
@@ -0,0 +1,34 @@
+# Python Using Pip
+
+We had seen about how to use `import` statements to `import` various modules and to use them in our programs. Python itself comes with several built-in modules. But python community has more to offer.
+> Its the modules that makes python so powerful!
+
+Third party module has added so much more functionality to Python. Now we would learn how to install these modules so that we can use those in our programs.
+
+The simplest way is to use `pip`
+
+```
+pip install <module_name>
+```
+If you have used `npm`, then you can think of it as *npm* of python :smiley:.
+
+Last time, in [`import-statements`](#) wiki we used `requests` module as an example. As it is a third party module we have to install it separately after installing python.
+
+Installing it would be as simple as ```pip install requests``` . You can even pass various arguments along with it. The one that you'll come across more often is `--upgrade`.
+You can  upgrade a python module by :
+
+```
+pip install <module_name> --upgrade
+```
+ So for example to upgrade the requests module to its latest version would be as simple as `pip install requests --upgrade`.
```

Comment:
~~So for example to upgrade the requests module to its latest version would be as simple as `pip install requests --upgrade`.~~ For example, to upgrade the requests module to its latest version would be as simple as `pip install requests --upgrade`.